src/main/java/org/kohsuke/github/Requester.java

```diff
@@ -497,73 +497,81 @@ public InputStream fetchStream() throws IOException {
     }
 
     private <T> T _fetch(String tailApiUrl, URL url, SupplierThrows<T, IOException> supplier) throws IOException {
-        while (true) {// loop while API rate limit is hit
+        int responseCode = -1;
+        String responseMessage = null;
+
```

Comment:
It is better to view this in split diff view.  Two methods got squashed down and blended.